crates/ruff_python_formatter/src/comments/format.rs

```diff
@@ -374,3 +398,18 @@ impl Format<PyFormatContext<'_>> for FormatEmptyLines {
         }
     }
 }
+
+/// Helper for measuring text.
+fn measure_text(text: &str, tab_width: u32) -> u32 {
```

Comment:
Nit: Can we pass `TabWidth` here. Makes the calling code cleaner and enforces that we indeed pass a tab width.

```suggestion
fn measure_text(text: &str, tab_width: TabWidth) -> u32 {
```

Reply:
Ty for the feedback. I kept swapping between `u32` and `TabWidth` lol. [`diff`](https://github.com/astral-sh/ruff/compare/64ee0d62ea9d02a5b9fd1a8c14ec6b13a16e7821..cfe95874d1e7a33b7e3b5ccfe59efb7b0af39bd9)